Dynatrace/dynatrace-operator
# Set logger in E2E test

## Description

Set a proper logger for controller-runtime in support archive E2E tests. Otherwise, the E2E logs are spammed with messages about a missing logger.

## How can this be tested?

Run E2E tests and check that output does **NOT** contain messages like:
```
[controller-runtime] log.SetLogger(...) was never called; logs will not be displayed.
```

## Checklist
- [n/a] Unit tests have been updated/added
